app/views/hiring_staff/vacancies/_show.html.haml

```diff
@@ -118,7 +118,7 @@
             %dd= @vacancy.newly_qualified_teacher
 
           %dt= t('jobs.expires_on')
-          %dd= format_date(@vacancy.expires_on)
+          %dd= format_date(@vacancy.expires_on) + (@vacancy.expiry_time.nil? ? '' : t('jobs.time_at') + format_time(@vacancy.expiry_time))
```

Comment:
Maybe extract out a method into the `VacancyPresenter` to allow it to be tested?

Reply:
Resolved this in e543ec7